src/VisualStudio/Core/Test/DebuggerIntelliSense/CSharpDebuggerIntellisenseTests.vb

```diff
@@ -459,6 +459,162 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.DebuggerIntelliSense
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
+    Data: {System.Collections.ListDictionaryInternal}
+    HResult: -2146233088
+    HelpLink: null
+    InnerException: null
+    Message: "Exception of type 'System.Exception' was thrown."
+    Source: null
+    StackTrace: null
+    TargetSite: null
+$$</Document>
+                               <Document>class Program
+{
+    static void Main(string[] args)
+    [|{|]
```

Comment:
This is supposed to be a selected open curly brace, right? I'm concerned that the test file markup parser will consume the [|, then consume the {|, then never find the closing |] because the | was already consumed. Can you confirm this is doing what you expect?